examples/research_projects/unilm/layoutlmv2/README.md

```diff
@@ -0,0 +1,116 @@
+<!---
+Copyright 2020 The HuggingFace Team. All rights reserved.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+-->
+
+# LayoutLMv2
+
+## Using Huggingface Trainer
+
+Fine-tuning the LayoutLMv2 library model. The main script `run_layoutlmv2.py` leverages the 🤗 Datasets library and the Trainer API. You can easily
+customize it to your needs if you need extra processing on your datasets.
+
+It will either run on a datasets hosted on our [hub](https://huggingface.co/datasets) or with your own text files for
+training and validation, you might just need to add some tweaks in the data preprocessing.
```

Comment:
"your own text files" is maybe not that appropriate, because the FUNSD dataset for example consists of annotated document images, each of which should contain a list of words + corresponding bounding boxes and NER tags.